worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -293,11 +304,11 @@ private IBaseBundle getBundle(String contractNumber, int searchMonth, int search
      * @return if we're in sandbox, return the synthetic data year unless it's the new Synthea data which can use
      * the correct year
      */
-    int getCorrectedYear(Contract contract, int coverageYear) {
+    int getCorrectedYear(Optional<Contract> optionalContract, int coverageYear) {
```

Comment:
Having an optional as input to a function is really weird.

Reply:
I didnt want to make a isPresent check in a constructor class. Passing and making the check here, then passing an int to the above class is less weird.